pkg/proxy/iptables/proxier_test.go

```diff
@@ -1415,6 +1414,10 @@ func (tracer *iptablesTracer) ruleMatches(rule *iptablestest.Rule, sourceIP, des
 		}
 	}
 
+	if rule.Protocol != nil && !rule.Protocol.Matches(protocol) {
```

Comment:
uppps :/

Reply:
glad you find it

Reply:
(It wasn't a bug really, it was just a feature I hadn't bothered to implement before because it didn't really matter much.)